src/userscores/qml/MuseScore/UserScores/ExportDialog.qml

```diff
@@ -0,0 +1,237 @@
+                        Layout.fillWidth: true
+
+                        spacing: 12
+
+                        Column {
+                            Layout.fillWidth: true
+                            Layout.preferredHeight: childrenRect.height
+
+                            spacing: 6
+
+                            StyledTextLabel {
+                                Layout.fillWidth: true
+                                text: qsTrc("userscores", "Export To:")
+                                font.capitalization: Font.AllUppercase
+                            }
+
+                            Row {
+                                id: exportToPath
+
+                                Layout.fillWidth: true
+                                Layout.preferredHeight: childrenRect.height
+
+                                spacing: 6
+
+                                TextInputField {
+                                    id: fileExportPathInput
+
+                                    height: 30
+                                    width: rightRectangle.width * 0.8
+
+                                    currentText: scoresModel.getExportPath()
+
+                                    onCurrentTextEdited: {
+                                        scoresModel.setExportPath(newTextValue);
+                                    }
+                                }
+
+                                FlatButton {
+                                    id: browsePathButton
+
+                                    icon: IconCode.NEW_FILE
+
+                                    height:30
+                                    width: rightRectangle.width * 0.1
+
+                                    onClicked: {
+                                        fileExportPathInput.currentText = scoresModel.chooseExportPath();
+                                    }
+                                }
+
+                            }
+                        }
+
+
+                        Column {
+                            Layout.fillWidth: true
+                            Layout.preferredHeight: childrenRect.height
+
+                            spacing: 6
+
+                            StyledTextLabel {
+                                Layout.fillWidth: true
+                                Layout.topMargin: 24
+
+                                text: qsTrc("userscores", "Export As: ")
+                                font.capitalization: Font.AllUppercase
+
+                            }
+
+                            StyledComboBox {
+                                id: fileTypeSelect
+
+                                property var currValue: 1
+
+                                width: rightRectangle.width * 0.8
+
+                                textRoleName: "text"
+                                valueRoleName: "value"
+
+                                model: [
+                                    { text: "PDF File", value: 1, suffix: "pdf" },
```

Comment:
The current list of available export types is hardcoded in qml
You should take a list of registered writers and all the information(title, suffix) about writer from WritersRegister. Then if we add a new writer, it will automatically appear in the list